packages/sdk/src/actions/cancelOrder.ts

```diff
@@ -3,27 +3,32 @@ import { Signer } from 'ethers'
 import { executeSteps } from '../utils'
 import { getClient } from '.'
 
-type CancelOrderPathParameters =
-  paths['/execute/cancel/v2']['get']['parameters']['query']
+type CancelOrderBodyParameters =
+  paths['/execute/cancel/v3']['post']['parameters']['body']
 
-export type CancelOrderOptions = Omit<CancelOrderPathParameters, 'maker' | 'id'>
+export type CancelOrderOptions = Omit<
+  NonNullable<CancelOrderBodyParameters['body']>,
+  'orderIds'
+>
 
 type Data = {
-  id: CancelOrderPathParameters['id']
+  id?: string
+  ids?: string[]
   signer: Signer
   options?: CancelOrderOptions
   onProgress: (steps: Execute['steps']) => any
 }
 
 /**
- * Cancel an offer or listing
+ * Cancel offers or listings
  * @param data.id Id of the order to cancel
+ * @param data.ids Ids of the orders to cancel
```

Comment:
I would prefer if we just have one interface of ids here, if you just need to pass one then simply pass an array of one:
`[id]`
This will keep things consistent with the other actions.

Reply:
I was aiming to avoid any breaking change.

Reply:
It's ok to make a breaking change. We can just make sure to deploy it with as a minor instead of a patch version.

Reply:
You'll also need to update the ui package to pass in an array with 1 id.
https://github.com/search?q=repo%3Areservoirprotocol%2Freservoir-kit+cancelOrder%28&type=code

Reply:
Done, would be good to hear from @georgeroman regarding the status of `/execute/cancel/v3`. I figured that since it was documented [here](https://docs.reservoir.tools/reference/postexecutecancelv3) that it wouldn't be changing anymore. I know things are rapidly changing / evolving. I'm curious what my expectations for things should be.